singleScraper/dockerfile

```diff
@@ -0,0 +1,11 @@
+# Start with a base image containing Java runtime
```

Comment:
WOW....

Reply:
Just wrong, nevertheless it seems that its not that easy to run pupperteer inside a docker.